--- /etc/speech-dispatcher/speechd.conf	2026-09-24 04:22:11.706476667 -0700
+++ /var/home/user/.config/speech-dispatcher/speechd.conf	2026-09-24 05:24:05.310106458 -0700
@@ -269,7 +269,9 @@
 #    or ~/.config/speech-dispatcher) or absolute
 
 #AddModule "espeak"                   "sd_espeak"    "espeak.conf"
-AddModule "espeak-ng"                "sd_espeak-ng" "espeak-ng.conf"
+#AddModule "espeak-ng"                "sd_espeak-ng" "espeak-ng.conf"
+#AddModule "kokoro"                   "sd_generic"   "kokoro.conf"
+AddModule "kokoro"                   "sd_kokoro"    "kokoro.conf"
 #AddModule "festival"                 "sd_festival"  "festival.conf"
 #AddModule "flite"                    "sd_flite"     "flite.conf"
 #AddModule "ivona"                    "sd_ivona"     "ivona.conf"
@@ -298,7 +300,7 @@
 # The DefaultModule selects which output module is the default.  You
 # must use one of the names of the modules loaded with AddModule.
 
-# DefaultModule espeak-ng
+DefaultModule kokoro
 
 # The LanguageDefaultModule selects which output modules are preferred
 # for specified languages.
